### DISCUSSION TITLE
Hello I have to implementation Wallet application and connect the dapp as QR code less

### DISCUSSION THREAD

**Discussion by KittinutzMuza** [WalletConnect/walletconnect-monorepo#2979] [Developer Support] [answered]
I have to implementation Wallet application and connect the dapp as QR code less. 
My project implement via React native with Expo. 
By Given user open DAPP via In my application  browser 
And when press connect will show the modal to approve or reject connect dapp.
please provide the example.

**Comment by boidushya** [comment]
Hey, please refer to our [React Native Examples](https://github.com/WalletConnect/react-native-examples)

**Reply by KittinutzMuza** [reply]
I look all example I want my user open dapp via webview and click connect then make connect with my wallet

**Comment by arbaz-meo** [comment, marked as answer]
@KittinutzMuza  i got you . 

you can use sign client  and  genrate the qr code by getting wc uri . simmilar you can connect your dapp on mobile by using deep link
